packages/pilot/src/containers/RecipientDetails/Info/index.js

```diff
@@ -0,0 +1,153 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import Tree from 'react-json-tree'
+
+import {
+  CardContent,
+  CardSection,
+  CardTitle,
+} from 'former-kit'
+
+import { pick } from 'ramda'
+
+import PartnerInfo from '../../AddRecipient/ConfirmStep/PartnerInfo'
+import ReceiverInfo from '../../AddRecipient/ConfirmStep/ReceiverInfo'
+
+import styles from './style.css'
+
+const RecipientDetailInfo = ({
+  identification,
+  metadata,
+  t,
+}) => (
+  <Fragment>
+    <CardContent>
+      <CardSection>
+        <CardTitle className={styles.title} title={t('pages.recipient_detail.recipient_information')} />
+        <CardContent>
+          <ReceiverInfo
+            identification={pick([
+              'cnpj',
+              'cnpjEmail',
+              'cnpjInformation',
+              'cnpjName',
+              'cnpjPhone',
+              'cnpjUrl',
+              'cpf',
+              'cpfEmail',
+              'cpfInformation',
+              'cpfName',
+              'cpfPhone',
+              'cpfUrl',
+              'documentType',
+            ], identification)}
+            t={t}
+          />
+        </CardContent>
+      </CardSection>
+    </CardContent>
+    <CardContent>
+      <CardSection>
+        <CardTitle className={styles.title} title={t('pages.recipient_detail.partner_information')} />
+        <CardContent>
+          <PartnerInfo
+            identification={pick([
+              'partnerNumber',
+              'partner0',
+              'partner1',
+              'partner2',
+              'partner3',
+              'partner4',
+            ], identification)}
+            t={t}
+          />
+        </CardContent>
+      </CardSection>
+    </CardContent>
+    <CardContent>
+      <CardSection>
+        <CardTitle className={styles.title} title={t('pages.recipient_detail.metadata')} />
```

Comment:
Aqui pode-se quebrar estas props em multiplas linhas.
```suggestion
        <CardTitle
          className={styles.title}
          title={t('pages.recipient_detail.metadata')}
        />
```